### PR TITLE
package: upgrade patternfly 4 to new major release

### PR DESCRIPTION
**Pull request by jkozol**
Patternfly 4 is upgraded. The patternfly provdided script pf-codemods was applied to fix component and property name changes. This will remain a draft PR until all elements have visual parity with the old version of patternfly 4.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "2.71.7",
```
Comment (KKoukiou) [inline package.json:10]: This needs update as well.

File: main.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ import { Provider } from "react-redux";
 import "@patternfly/patternfly/patternfly-addons.css";
 import "@patternfly/patternfly/layouts/Flex/flex.css";
 import "@patternfly/patternfly/utilities/Display/display.css";
+import "@patternfly/react-core/dist/styles/base.css";
```
Comment (KKoukiou) [inline main.js:19]: How is this change related to PF4 version upgrade?

Comment (jkozol) [reply main.js:19]: This is from the instructions provided by the `react-core` npm package [here](https://www.npmjs.com/package/@patternfly/react-core):

>  In order to have consistent styles throughout the application, PatternFly Base CSS should be used in the project, or some components may diverge in appearance. This import statement should be placed before your main application component is imported.
> 
> `import '@patternfly/react-core/dist/styles/base.css';`

**Comment by jkozol** [comment]
@jgiardino I combined your pagination fixes together.

File: main.js (hunk 2 above)
Comment (jkozol) [reply main.js]: Fixed
